crates/net/network/src/peers/manager.rs

```diff
@@ -579,6 +579,7 @@ impl PeersManager {
     fn fill_outbound_slots(&mut self) {
         // as long as there a slots available try to fill them with the best peers
         while self.connection_info.has_out_capacity() {
+            let current = Instant::now()
```

Comment:
here we need to call the tick function that takes/gets the `current` instant, loops over connected peers, updates the peer's reputation based on `current - last_tick`, updates `last_tick` to `current`

the reputation function should be something that's `f(seconds) = `f((current - last_tick).as_secs())``

Reply:
Okay sir